README.md

```diff
@@ -50,6 +50,21 @@ This gem provides a simple script, that can help you with this task:
 4. Run the script (may take a while, when you want to continue working on your project meanwhile run this in a separate checkout): `rubocop-autofix`
 5. Review all commits made by the script and run your tests. You can now drop certain commits of cops you don't want. Often it may make some sense to run the script again with changed settings, since rebasing 100+ commits is no fun.
 
+# Development
+
+Any contributions are welcome. If you attempt to change the behavior of this gem it might be wise to open an issue first to discuss the change. Otherwise feel free to open a PR.
+
+Every PR should have a change in the CHANGELOG file (within the `master` section) briefly outlining the attempted changes.
+
+## Release a new version
+
+To release a new version, follow these steps:
+
+* update CHANGELOG to reflect that new release
+* update the version in `rubocop-bitcrowd.gemspec` according to [semver](https://semver.org/)
+* commit that change and tag it with a `vMAJOR.MINOR.PATH` title. Ideally, the tag-body has the corresponding CHANGELOG part.
```

Comment:
For the sake of simplicity, I'd suggest to add the copy-paste (& adapt) ready commands to create a tag here.